src/_pytest/terminal.py

```diff
@@ -545,13 +546,19 @@ def pytest_runtest_logstart(
             self.write_fspath_result(nodeid, "")
             self.flush()
 
+    class TestStatus(NamedTuple):
```

Comment:
Can you please add a docstring here, also describing each member?

Reply:
I suggest we do not declare this class inside `TerminalReporter`, but make it top-level and also available directly from `pytest.TestStatus`.

Also not sure about the name itself, `TestStatus` feels too generic and similar to `TestReport` (although I see where it is named after)... however no good suggestions come to mind either, bringing this up in case someone has a good suggestion.

Reply:
the name may need a litte bikeshedding as this is bacsically the terminalwriter summary test status

Reply:
@nicoddemus, just a question on `reference.rst`, how should I be documenting this new structure in there? I found [this section](https://github.com/pytest-dev/pytest/blob/main/doc/en/reference/reference.rst#reporting-hooks) but am not too sure what to add, something like:
```
.. class:: TestStatus
```

As for the name, I'm all ears, I just went for what seemed a good fit with the `pytest_report_teststatus` hook (as you noticed already). Any suggestions?

Reply:
As for the name, how about `TestShortLogReport`?